network/p2p/gossip/gossip.go

```diff
@@ -34,6 +34,17 @@ const (
 	unsentType = "unsent"
 	sentType   = "sent"
 
+	// omitted indicate that the gossipable element was not added to the set due to some reason.
+	// for sent message, we'll use notReceive below.
+	omittedLabel = "omitted"
+
+	omittedFailedVerification = "failed-verification"
+	omittedDuplicate          = "duplicate"
+	omittedMalformed          = "malformed"
+	omittedOther              = "other"
+	omittedNot                = "not"
+	notReceive                = "n/a"
```

Comment:
I'm surprised this works... I didn't think prometheus allowed special characters other than colons/underscores.

Reply:
changed to "not_receive" to avoid potential issues.